palette/examples/color_scheme.rs

```diff
@@ -93,8 +93,8 @@ fn main() {
         .and_then(|r| r.parse().ok())
         .expect("the blue channel must be a number in the range [0-255]");
 
-    let primary: Color = Srgb::new(red, green, blue)
-        .into_format()
+    let primary: Hsv = Srgb::new(red, green, blue)
```

Comment:
This should be `Lch` to achieve the same results.

Reply:
Ah yes, I already thought that it had to be the color type I used

Reply:
This kind of illustrates one of the reasons why I wanted to remove it. The selling point for `Color` was that it would pick the most suitable color space for the task. The downside was that it would hide this choice and could produce surprising results.